package.json

```diff
@@ -59,6 +59,7 @@
     "react-hook-form": "^7.51.3",
     "react-is": "^18.2.0",
     "react-merge-refs": "^2.1.1",
+    "react-popper": "^2.3.0",
```

Comment:
we're using floating-ui elsewhere, which is the successor to popper. even though the headless docs say to use popper, you can probably make this work with floating-ui

Reply:
Example here: https://github.com/tailwindlabs/headlessui/blob/main/playgrounds/react/pages/menu/menu-with-floating-ui.tsx

Linked from here: https://github.com/tailwindlabs/headlessui/issues/2780